Packs/Wiz/README.md

```diff
@@ -8,4 +8,4 @@ Use this Content Pack to prioritize risks that span your entire cloud estate and
 
 _For more information, visit [Wiz Inc.](https://wiz.io/)_
 
-[![Wiz Overview](https://raw.githubusercontent.com/demisto/content/master/Packs/Wiz/Author_image.png)](https://wiz.io)
\ No newline at end of file
+[![Wiz Overview](Author_image.png)](https://wiz.io)
```

Comment:
Is this the right link?

Reply:
Yes, you can see its display when viewing the file.